tests/Feature/Project/ShowProjectTest.php

```diff
@@ -21,7 +21,27 @@ public function a_logged_in_user_can_view_details_of_a_project()
 
         $response->assertSuccessful();
 
-        $response->assertSee($project->name);
+        // escape the apostrophe in the same manner as blade is
+        $response->assertSee(e($project->name));
+
+        $response->assertSee($project->description);
+        $response->assertSee($project->client->name);
+    }
+
+    /** @test */
+    public function a_logged_in_user_can_view_details_of_a_project_with_an_apostrophe_in_the_name()
```

Comment:
Test case FTW 💪

Reply:
Just to expand in this. The reason it was failing was because we were checking that the project name with an apostrophe was in the response content. But the apostrophe had been encoded so wasn't there to be found. This now checks for the encoded version of the project name.

Reply:
Also, that build failure (https://github.com/YakTrack/YakTrack/pull/146/checks?check_run_id=1244386965) looks like the same issue, just in a different test.